commercetools-models/src/test/java/io/sphere/sdk/products/search/ProductProjectionSearchMainIntegrationTest.java

```diff
@@ -25,13 +31,37 @@
 import static java.util.stream.Collectors.toList;
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Ignore
+
 public class ProductProjectionSearchMainIntegrationTest extends ProductProjectionSearchIntegrationTest {
 
     public static final NamedAttributeAccess<String> COLOR_ATTRIBUTE_ACCESS = AttributeAccess.ofString().ofName(ATTR_NAME_COLOR);
     public static final NamedAttributeAccess<Long> SIZE_ATTRIBUTE_ACCESS = AttributeAccess.ofLong().ofName(ATTR_NAME_SIZE);
     public static final ProductProjectionFilterSearchModel PRODUCT_MODEL = ProductProjectionSearchModel.of().filter();
 
+    @Test
+    public void selectAProductByLocaleProjectionInProductProjectionSearch() {
+        final String localeProjection = "en-EN";
+        ProductFixtures.withProduct(client(), product -> {
+            final ProductProjectionSearch searchRequest = ProductProjectionSearch.ofStaged()
+                    .withQueryFilters(m -> m.id().is(product.getId()))
+                    .withLocaleSelection(LocaleSelection.of(localeProjection));
+            assertThat(searchRequest.httpRequestIntent().getBody().toString()).contains("localeProjection=%5Ben-EN%5D");
+        });
+    }
+
+    @Test
+    public void selectAProductByStoreProjectionInProductProjectionSearch() {
+        BlockingSphereClient client = client();
+        StoreFixtures.withStore(client, store -> {
+            ProductFixtures.withProduct(client, product -> {
+                final ProductProjectionSearch searchRequest = ProductProjectionSearch.ofStaged()
+                        .withQueryFilters(m -> m.id().is(product.getId()))
+                        .withStoreSelection(StoreSelection.of(store.getKey()));
+                assertThat(searchRequest.httpRequestIntent().getBody().toString()).contains("storeProjection=" + store.getKey());
```

Comment:
```suggestion                                                                
                assertThat(searchRequest.httpRequestIntent().getPath()).contains("storeProjection=" + store.getKey());
```

Reply:
no. For search requests the query is sent in the body of a post request. So this is correct. ;)

Reply:
I cannot apply this method here because it takes only the first part of the path `/product-projections/search`

Reply:
ah ok, i wasn't aware of it 😅